packages/api/src/controllers/loginController.js

```diff
@@ -36,11 +34,7 @@ class loginController extends baseController {
 
         delete data.password_hash;
 
-        const token = await jwt.sign(
-          { ...data },
-          process.env.JWT_SECRET,
-          { expiresIn: SECONDS_IN_A_WEEK },
-        );
+        const token = await createAccessToken({ ...data })
```

Comment:
If you're going to make this change here, you can feel free to make the same change in the sign up route (in userController)